src/Resources/app/storefront/src/swag-paypal.abstract-buttons.js

```diff
@@ -88,9 +91,13 @@ export default class SwagPaypalAbstractButtons extends SwagPayPalScriptBase {
      * @param {*} [error=undefined] - The error. Can be any type, but will be converted to a string
      */
     // eslint-disable-next-line no-unused-vars
-    onErrorHandled(code, fatal, error) {
+    onErrorHandled(code, fatal, error, isCheckout = false) {
         window.scrollTo(0, 0);
-        window.location.reload();
+        
+        // Make sure only relooad when the current page is checkout
+        if (isCheckout) {
+            window.location.reload();
```

Comment:
No flash, no need to scroll up to it:
```suggestion
        // Make sure only relooad when the current page is checkout
        if (isCheckout) {
            window.scrollTo(0, 0);
            window.location.reload();
```